reference-architecture/azure-ansible/infranode.json

```diff
@@ -118,7 +118,8 @@
               "uri" : "[concat('http://', parameters('sa'), '.blob.core.windows.net/openshiftvmachines/', parameters('vmName'), '.vhd')]"
             },
             "caching" : "ReadWrite",
-            "createOption" : "FromImage"
+            "createOption" : "FromImage",
+            "diskSizeGB" : "128"
```

Comment:
There multiple partitions here, and now a check for 60 Gig on root free. 128 is kind of the good place to be, as depending on storage type, your charged for 128 anyway.. Also it avoids downstream issues later.